mummer/4.0.0/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the following stage. We use `app` for the production image
+FROM ubuntu:xenial as app
```

Comment:
I would recommend upgrading to `ubuntu:focal` since `ubuntu:xenial` is EOL

Reply:
I did have a question about this. In Dockerhub the ubuntu image does have a latest tag; do we avoid using this for our Staph-b docker-builds because as ubuntu gets updated it could cause some incompatibilities ?

Reply:
Yes, `ubuntu:latest` will change a lot more frequently than `ubuntu:focal` will. Changes to `ubuntu:focal` will be restricted to that particular release of Ubuntu and will mostly include bug fixes and security patches (which we would welcome 👍 )